pkg/k8s/wrapper.go

```diff
@@ -220,3 +224,59 @@ func (k *k8sWrapper) ListEvents(ops []client.ListOption) (*eventsv1.EventList, e
 	}
 	return events, nil
 }
+
+func (k *k8sWrapper) GetCNINode(cniNodeName, cniNodeNamespace string) (*rcv1alpha1.CNINode, error) {
+	cninode := &rcv1alpha1.CNINode{}
+	if err := k.cacheClient.Get(k.context, types.NamespacedName{
```

Comment:
Check out client.ObjectKeyFromObject(cniNode)

Reply:
I am not sure why we need get the ObjectKey from the cninode after Get()? Can you elaborate your thought? Thanks